modules/api/src/main/java/org/apache/fluo/api/client/FluoAdmin.java

```diff
@@ -119,6 +119,13 @@ public TableExistsException() {
   void initialize(InitializationOptions opts)
       throws AlreadyInitializedException, TableExistsException;
 
+  /**
+   * Removes Fluo application, Accumulo table and shared configuration in Zookeeper. Shared configuration
+   * consists of all properties except those with
+   * {@value org.apache.fluo.api.config.FluoConfiguration#CONNECTION_PREFIX} prefix.
+   */
```

Comment:
Javadoc needs a `@since` tag.  I think we can get this into 1.2, so could do `@since 1.2.0`